v63005/u_inref/gtm5574.csh

```diff
@@ -0,0 +1,31 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+
+# Set to avoid wrapping from bc
```

Comment:
Replace with "# set to avoid line wrapping at 68 columns or so which breaks long numbers into multiple lines otherwise"